tap_jira/streams.py

```diff
@@ -115,7 +116,7 @@ def write_page(self, page):
 
 
 class Projects(Stream):
-    def sync(self):
+    def sync_on_prem(self):
```

Comment:
Do add function comment and reason for this name change.

Reply:
added comments